components/movementsensor/client_test.go

```diff
@@ -197,23 +204,23 @@ func TestClient(t *testing.T) {
 
 		_, _, err = client2.Position(context.Background(), make(map[string]interface{}))
 		test.That(t, err, test.ShouldNotBeNil)
-		test.That(t, err.Error(), test.ShouldContainSubstring, "can't get location")
+		test.That(t, err.Error(), test.ShouldContainSubstring, errLocation.Error())
```

Comment:
use `test.ShouldBeError` in this file as well? (I noticed you made the change elsewhere)

Reply:
Since the client tests come back as rpc errors test.ShouldBeError doesn't work